charts/registry-scanner/templates/secret.yaml

```diff
@@ -14,10 +14,10 @@ data:
   aws_region: {{ required "A valid .Values.config.aws.region is required" .Values.config.aws.region | b64enc | quote }}
   {{- else if eq .Values.config.registryType "gcr" }}
   registryUser: {{ "_json_key" | b64enc | quote }}
-  registryPassword: {{ required "A valid .Values.config.registryPassword is required" (include "registry-scanner.sanitizeJSON" (.Values.config.registryPassword | toJson)) | b64enc |quote }}
+  registryPassword: {{ required "A valid .Values.config.registryPassword is required" .Values.config.registryUser | b64enc | quote }}
   {{- else if eq .Values.config.registryType "gar" }}
   registryUser: {{ "_json_key_base64" | b64enc | quote }}
```

Comment:
btw, GAR registry supports both user types: _json_key and _json_key_base64. For _json_key we can provide not encoded json key and for _json_key_base64 we need to provide encoded one.

that's why I haven't touched this logic to let customer decide on what they want.